base_dj/tests/test_xmlid.py

```diff
@@ -54,9 +54,14 @@ def test_xmlid_no_specific_rule_no_name_field(self):
         rec = self.env['res.partner.bank'].create({'acc_number': '01234', })
         # no specific rule other than `__setup__` as prefix
         # and record ID as suffix (odoo's std)
-        self.assertEqual(
-            rec._dj_export_xmlid(),
-            '__setup__.res_partner_bank_{}'.format(rec.id)
+        self.assertTrue(
+            # XXX: use `startswith` since from this change
```

Comment:
Why XXX, is it still a TODO or a FIXME ?

I can see `startswith` already being used here. This comment line can probably be dropped.
Otherwise, I don't see why we need this comment as an highlighted comment.

Reply:
fixed w/ assertRegexpMatches :wink: